### ISSUE TITLE
"List of sites" page block with pagination

### ISSUE THREAD

**Issue by allanaaa** [omeka/omeka-s#2168]
<img width="689" alt="Screenshot 2024-03-19 at 13 14 31" src="https://github.com/omeka/omeka-s/assets/28012148/07798cf9-26dd-4463-8642-39b3cef51216">
<img width="639" alt="Screenshot 2024-03-19 at 13 21 15" src="https://github.com/omeka/omeka-s/assets/28012148/d8886700-aacd-4bc1-82fa-383933612448">

When pagination is turned on, the buttons appear odd and the block sometimes overflows its available width.

**Comment by kimisgold** [comment]
There should now be some corrective styles for list of sites pagination ready for a look.

**Comment by allanaaa** [comment]
No changes yet.

**Comment by kimisgold** [comment]
Hmm, can you do a hard refresh? I was able to see changes on your testing installation.

**Comment by allanaaa** [comment]
Yep, looks good in Foundation - Thanks Roy is still: 


<img width="1129" alt="Screenshot 2024-03-20 at 14 02 39" src="https://github.com/omeka/omeka-s/assets/28012148/69a2d527-acee-4d70-9cce-adbb37966f77">

**Comment by kimisgold** [comment]
I forgot that Foundation had some custom markup for pagination, so I went back and corrected the page block styles to reference the actual default markup. Foundation has also been adjusted accordingly. I think everything's ready for another look.

**Comment by allanaaa** [comment]
It looks better in Thanks Roy, but maybe some spacing work is needed on the buttons:

<img width="701" alt="Screenshot 2024-03-21 at 14 37 30" src="https://github.com/omeka/omeka-s/assets/28012148/0bcbbf75-8f26-4214-a299-979cbdc4d3f1">

**Comment by allanaaa** [comment]
Foundation looks good. 

Default is also a bit wonky in buttons and spacing:

<img width="918" alt="Screenshot 2024-03-21 at 14 39 25" src="https://github.com/omeka/omeka-s/assets/28012148/9f1a17e6-05f8-473a-8df8-82c355dd1324">

**Comment by allanaaa** [comment]
Center Row has different-sized buttons?

<img width="756" alt="Screenshot 2024-03-21 at 14 40 20" src="https://github.com/omeka/omeka-s/assets/28012148/27a39693-94fe-402a-8f62-dcf74522ef46">

**Comment by allanaaa** [comment]
And Cozy looks pretty much the way Default does:

<img width="818" alt="Screenshot 2024-03-21 at 14 40 55" src="https://github.com/omeka/omeka-s/assets/28012148/5fc867a5-11d1-4027-9465-b4a150615f5b">

**Comment by kimisgold** [comment]
I've added some more default styles to the core to bridge some of the inconsistencies. Papers, The Daily, Center Row, and Cozy also have updates. The other themes should be looking okay with the updated core styles.

**Comment by allanaaa** [comment]
Any chance this has messed up the pagination buttons in the admin side? 
<img width="633" alt="Screenshot 2024-03-25 at 15 20 13" src="https://github.com/omeka/omeka-s/assets/28012148/fcf9729e-3687-4ffe-aabc-9bbafbefabb5">
<img width="259" alt="Screenshot 2024-03-25 at 15 20 23" src="https://github.com/omeka/omeka-s/assets/28012148/a8f3ae3e-2c4c-4df8-a0ca-e64c148511db">

**Comment by kimisgold** [comment]
Yup, though an easy fix is in and ready for a look.

**Comment by allanaaa** [comment]
Admin is fixed. 

Thanks Roy could still use some refinement:
<img width="470" alt="Screenshot 2024-03-26 at 17 09 39" src="https://github.com/omeka/omeka-s/assets/28012148/d44e14f5-630f-4605-8133-cb9c80ef1349">

Default appears to have genuinely no text in the box. Not just a "1" in white or something. 
<img width="587" alt="Screenshot 2024-03-26 at 17 10 51" src="https://github.com/omeka/omeka-s/assets/28012148/4bd6ca81-d065-4856-866f-423bc0658f6c">

Everything else seems okay at the moment.

**Comment by kimisgold** [comment]
Improved spacing on both, and default had what i think was a typo on text inputs giving them massive padding on the sides that clipped the actual content. Should be fixed and ready for review.

**Comment by allanaaa** [comment]
Default looks great now. 

Thanks Roy is an improvement for sure - I'm not sure if you want the page box to be the same width as the thumbnails? But it's not quite there, and I don't know that it needs to be that big, really. 

<img width="824" alt="Screenshot 2024-04-01 at 12 11 30" src="https://github.com/omeka/omeka-s/assets/28012148/6c4591d2-0ff8-4c06-9290-2fc1cfe1eae5">

And the text is just the tiniest bit misaligned from each other (the "of 3" and the "1-2 of 5"), which I find really annoying.

<img width="547" alt="Screenshot 2024-04-01 at 12 13 28" src="https://github.com/omeka/omeka-s/assets/28012148/e240d242-7b7f-4bf5-9816-910e15be5676">

**Comment by kimisgold** [comment]
The text input is that width because there are many projects that have huge quantities of resources. It's somewhat arbitrary that we've made it about 7 digits wide, but I've reduced it to be about 5 digits wide. I'm hesitant to match it to the thumbnail width: the pagination is a reusable element, and I'm not inclined to have that width customized across all its different contexts. I've refined the text alignment a bit, and it should appear more consistently vertically aligned.
